plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -2112,7 +2114,7 @@ void producer_plugin_impl::process_scheduled_and_incoming_trxs( const fc::time_p
                        "[TRX_TRACE] Block ${block_num} for producer ${prod} is REJECTING scheduled tx: ${txid}, time: ${r}, auth: ${a} : ${details}",
```

Comment:
See if this can use the trx_log_result instead of duplicating very similar log output.

Reply:
Since transaction is not available, it feels to forced to try and make this work.